### DISCUSSION TITLE
推荐一款增加arco-design开发体验和幸福感的插件😁

### DISCUSSION THREAD

**Discussion by Simon-He95** [arco-design/arco-design#2346] [Show and tell]
只需要安装此插件和@acro-design/web-react，即可享受这样的提示效果，一个好的代码提示能让你心情愉悦，和减轻很多查阅文档的时间，和提高工作效率
![arco-design](https://github.com/arco-design/arco-design/assets/57086651/d631e4c0-d820-4a08-a559-cbbb65472d75)

**Comment by liucangsong** [comment]
VSCode  + Github Copilot 完美，注意使用面向业务的长变量名，多写注释进行投喂。机器人就会一片一片帮你写代码
